node/jest.config.js

```diff
@@ -3,6 +3,20 @@ module.exports = {
     transform: { "^.+\\.ts?$": "ts-jest" },
     testEnvironment: "node",
     testRegex: "/tests/.*\\.(test|spec)?\\.(ts|tsx)$",
-    moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
+    moduleFileExtensions: [
+        "ts",
+        "tsx",
+        "js",
+        "jsx",
+        "json",
+        "node",
+        "cjs",
+        "mjs",
+    ],
     testTimeout: 20000,
+    globals: {
+        "ts-jest": {
+            tsconfig: "tsconfig-mjs.json",
```

Comment:
Can you explain what you do here?

Reply:
explain in the file? a comment?
jest as a default looking for tsconfig.json file for ts configuration. since we dont have one anymore i gave it the updated file to use as a ts conf file.